GameServerLib/Logic/Game.cs

```diff
@@ -255,6 +257,15 @@ private bool HandleDisconnect(Peer peer)
                 {
                     PacketNotifier.NotifyUnitAnnounceEvent(UnitAnnounces.SummonerDisconnected, peerinfo.Champion);
                 }
+                _logger.LogCoreInfo("somebody disconnected");
+                disconnectedPlayers++;
+                var number = _playerManager.GetPlayers().Count - disconnectedPlayers;
+                _logger.LogCoreInfo(number.ToString());
+                if (number == 0)
+                {
+                    _logger.LogCoreInfo("no players connected");
+                    Program.SetToExit();
```

Comment:
1. Logging number of players connected seems unnecessary here.
2. This should have a delay.
3. This should be optional